src/recipe/commands/handlers/createRecipe.handler.ts

```diff
@@ -27,6 +29,10 @@ export class CreateRecipeHandler
     const file = await this.fileService.findFileById(recipeFromDb.fileId);
     const fileUrl = await this.s3Service.generatePresignedUrl(file.key);
 
-    return new RecipeResponse(recipeFromDb, fileUrl);
+    const recipeToReturn = new RecipeResponse(recipeFromDb, fileUrl);
+
+    await this.webhookService.sendWebhookWithRecipe(recipeToReturn);
```

Comment:
you have added logic to the existing method (createRecipe). You should add a test caste that would cover that. 
You should check if the proper method od webhookService was called